src/dune_sexp/syntax.mli

```diff
@@ -71,6 +75,8 @@ val create
 (** Return the name of the syntax. *)
 val name : t -> string
 
+val desc : unit -> (Loc.t * string, _) Decoder.parser
```

Comment:
Can you add the type of `_` here as well? `_` is useful for writing code but looking at an `mli` file `_` is really unhelpful. Also a short docstring would be appreciated.